src/pretalx/cfp/flow.py

```diff
@@ -369,13 +369,21 @@ def is_applicable(self, request):
             return self.event.questions.all().exists()
         return self.event.questions.exclude(
             Q(target=QuestionTarget.SUBMISSION)
-            & (~Q(tracks__in=[info_data.get("track")]) & Q(tracks__isnull=False))
+            & (
+                (~Q(tracks__in=[info_data.get("track")]) & Q(tracks__isnull=False))
+                | (
+                    ~Q(submission_types__in=[info_data.get("submission_type")])
+                    & Q(submission_types__isnull=False)
+                )
+            )
         ).exists()
 
     def get_form_kwargs(self):
         result = super().get_form_kwargs()
+        info_data = self.cfp_session.get("data", {}).get("info", {})
         result["target"] = ""
-        result["track"] = self.cfp_session.get("data", {}).get("track")
+        result["track"] = info_data.get("track")
+        result["submission_type"] = info_data.get("submission_type")
```

Comment:
Hm, is this correct? Previously we used ``self.cfp_session["data"]["track"]``, and now we're using ``self.cfp_session["data"]["info"]["track"]``.

Reply:
I tested it multiple times with different types of questions. As long as I am testing correctly, it seems to work 😅.

I believe there was a bug here. I think that there reason it seemed to work is because `is_applicable()` would hide the questions page entirely so nobody saw the bug. But, it is my assumption and I am not 100% sure.

Can you please check this on your end?